platforms.c

```diff
@@ -236,6 +237,19 @@ void pv_platform_add_condition(struct pv_platform *p, struct pv_condition *c)
 	}
 }
 
+void pv_platform_add_driver(struct pv_platform *p, plat_driver_t type, char *value)
+{
+	struct pv_platform_driver *d = calloc(1, sizeof(struct pv_platform_driver));
```

Comment:
where are we freeing up this?

Reply:
should be fixed now. please check @anibalportero